test/remote/gateways/remote_vpos_test.rb

```diff
@@ -0,0 +1,92 @@
+require 'test_helper'
+
+class RemoteVposTest < Test::Unit::TestCase
+  def setup
+    @gateway = VposGateway.new(fixtures(:vpos))
+
+    @amount = 100000
+    @credit_card = credit_card('5418630110000014', month: 8, year: 2021, verification_value: '258')
```

Comment:
Did these test card details come from vPOS specifically? Having the hard coded expiration will cause a hiccup when it expires, but it might just be the way it is if they gave these deets.

Reply:
Yes, vPOS specifically requires this expiration date. I'll set a reminder to check the test suite in August, though :D